game/src/test/scala/io/aigar/game/ResourcesSpec.scala

```diff
@@ -66,4 +88,31 @@ class ResourcesSpec extends FlatSpec with Matchers {
 
     cell.mass should equal(30)
   }
+
+  it should "returns a list of scoreMessages from multiple positions and players" in {
```

Comment:
`returns` => `return`?

Also, could we include in the description that this is on collision?
E.g.
`return a list of ScoreMessages for players that collided`